WordPress/Classes/ViewRelated/Me/All Domains/View Models/AllDomainsListViewModel+Strings.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+extension AllDomainsListViewModel {
+
+    enum Strings {
+        static let emptyStateTitle = NSLocalizedString(
+            "domain.management.default.empty.state.title",
+            value: "You don't have any domains",
+            comment: "The empty state title in All Domains screen when the user doesn't have any domains"
+        )
+        static let emptyStateDescription = NSLocalizedString(
+            "domain.management.default.empty.state.description",
+            value: "Tap the button below to add a new domain",
+            comment: "The empty state description in All Domains screen when the user doesn't have any domains"
+        )
+        static let emptyStateButtonTitle = NSLocalizedString(
+            "domain.management.default.empty.state.button.title",
+            value: "Add a domain",
```

Comment:
To match the designs
Ref: rvVgXupiyONUcJMgPSoMFj-fi-2928_12681
```suggestion
            value: "Find a domain",
```